packages/landing/scripts/bundle.js

```diff
@@ -99,8 +105,31 @@ async function bundleHtml(opts) {
     await fs.writeFile('./dist/index.html', htmlOutput);
 }
 
+async function recurseStatic(subDir = '') {
+    const srcDir = './static' + subDir;
+    const files = await fs.readdir(srcDir);
+    for (const fileName of files) {
+        const filePath = `${subDir}/${fileName}`;
+
+        console.log(`DEBUG SPECIAL_FILES[filePath]`, SPECIAL_FILES[filePath], filePath);
+        if (SPECIAL_FILES[filePath]) continue;
+
+        const srcPath = `${srcDir}/${fileName}`;
+        const destPath = `./dist${filePath}`;
+        const stat = await fs.stat(srcPath);
+        if (stat.isDirectory(srcPath)) {
+            await fs.mkdir(destPath);
+            await recurseStatic(filePath);
+            continue;
+        }
+        await fs.writeFile(destPath, await fs.readFile(srcPath));
+    }
+}
+
 async function main() {
+    await fs.rmdir('dist', { recursive: true });
```

Comment:
I dont think we want to delete this here.

I believe there is `clean` if you want to clean it.

Reply:
I think a bundle should clear the directory because there could be leftover files there that are no longer in `static`. I'm okay with not though if you prefer not to.

Reply:
All the other bundlers I use, webpack, parcel etc, have a very specific "clean" function if things are being deleted.

The static folder in theory is only deployed from a clean state so it shouldnt normally matter, and if you are doing a deployment you really want to `clean` the entire repo.

TLDR: yes please take it out.